dev/com.ibm.ws.microprofile.config.1.1_fat/fat/src/com/ibm/ws/microprofile/config/fat/tests/LibertySpecificConfigTests.java

```diff
@@ -0,0 +1,111 @@
+/*******************************************************************************
+ * Copyright (c) 2016, 2018 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.microprofile.config.fat.tests;
+
+import java.io.File;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.FileAsset;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+import com.ibm.ws.microprofile.appConfig.cdi.libertyTests.LibertyBuiltInConverterTestServlet;
+import com.ibm.ws.microprofile.appConfig.cdi.libertyTests.LibertyFieldTestServlet;
+import com.ibm.ws.microprofile.appConfig.classLoaders.test.libertyTests.LibertyClassLoadersTestServlet;
+import com.ibm.ws.microprofile.appConfig.converters.test.libertyTests.LibertyConvertersTestServlet;
+import com.ibm.ws.microprofile.appConfig.defaultSources.tests.libertyTests.LibertyDefaultSourcesTestServlet;
+import com.ibm.ws.microprofile.config.fat.repeat.RepeatConfigActions;
+import com.ibm.ws.microprofile.config.fat.repeat.RepeatConfigActions.Version;
+import com.ibm.ws.microprofile.config.fat.suite.SharedShrinkWrapApps;
+
+import componenttest.annotation.Server;
+import componenttest.annotation.TestServlet;
+import componenttest.annotation.TestServlets;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.custom.junit.runner.Mode;
+import componenttest.custom.junit.runner.Mode.TestMode;
+import componenttest.rules.repeater.RepeatTests;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.utils.FATServletClient;
+
+/**
+ * These tests test Config functionality specific to Liberty's implementation (mpConfig 1.x) of MicroProfile Config specification.
+ *
+ * For mpConfig versions <2.0, Open Liberty implemented some extra pieces of functionality beyond the MicroProfile Config specification.
+ *
+ * In additon, some tests check for error messages defined by Liberty, and others check functionality which changed between mpConfig 1.4 -> 2.0.
```

Comment:
Please list what these tests for.

Reply:
Any tests within `LibertySpecificConfigTests.java` classes are tests for functionality specific to Liberty's implementation (mpConfig < 2.0) of MP Config (as explained in the comment).

The tests should be self-explanatory, or have comments explaining their purpose.